src/layouts/main/styles/functions.js

```diff
@@ -0,0 +1,68 @@
+import {css} from 'styled-components';
+import {drawerWidth} from './constants';
+
+/**
+ * Returns drawer styles when is open.
+ *
+ * @param {object} theme
+ * @return {css} drawer styles when opened
+ */
+const getOpenDrawerStyles = (theme) => css`
+  width: ${drawerWidth}px;
+  transition: ${theme.transitions.create('width', {
+    easing: theme.transitions.easing.sharp,
+    duration: theme.transitions.duration.enteringScreen,
+  })};
+`;
+
+/**
+ * Returns drawer styles when is closed.
+ *
+ * @param {object} theme
+ * @return {css} drawer styles when closed
+ */
+const getClosedDrawerStyles = (theme) => css`
+  transition: ${theme.transitions.create('width', {
+    easing: theme.transitions.easing.sharp,
+    duration: theme.transitions.duration.leavingScreen,
+  })};
+  overflow-x: hidden;
+  width: ${theme.spacing(9)}px;
+
+  .MuiListItemText-root {
+    visibility: hidden;
+  }
+`;
+
+/**
+ * Returns drawer styles according to
+ * the opened state
+ *
+ * @param {{open: boolean, theme: object}} props
+ * @return {css} styles according to open state
+ */
+const getToggleDrawerStyles = ({open, theme}) => {
```

Comment:
nit: maybe just getDrawerStyles? Since you're not actually toggling

Reply:
Done!